torch/testing/_internal/common_methods_invocations.py

```diff
@@ -239,6 +240,17 @@ def __init__(self,
         self.aten_name = aten_name if aten_name is not None else name
         self.variant_test_name = variant_test_name
 
+        # Attribute to verify dynamic_dtypes are used.
+        self.dynamic_dtypes = any(map(lambda dtypes: isinstance(
+            dtypes, opinfo_helper._dynamic_dispatch_dtypes), (dtypes, dtypesIfCUDA)))
```

Comment:
Should the latter set include dtypesIfCPU (until we remove it in favor of dtypes) and dtypesIfROCm, too?

Reply:
We can leave out `dtypesIfROCM` (as it is mostly consistent with `dtypesIfCUDA` which we cover.

Not really sure about `dtypesIfCPU` as it is seldom used (but we explicitly support `cpu` device type)

Reply:
Might as well add dtypesIfCPU and dtypesIfROCM for now, just in case